lib/server-info-route-handler.js

```diff
@@ -12,14 +12,13 @@ const debug = process.env.KOOP_LOG_LEVEL === 'debug' || process.env.LOG_LEVEL ==
 
 function serverMetadata (json, { query = {} } = {}) {
   const { extent, metadata, ...rest } = json
-  const { maxRecordCount, hasStaticData, description } = { ...metadata, ...rest }
+  const { maxRecordCount, hasStaticData, description, copyrightText } = { ...metadata, ...rest }
   const spatialReference = getSpatialReference(json, query)
   const { layers, tables, relationships } = normalizeInputData(json)
   const fullExtent = getServiceExtent({ extent, metadata, layers, spatialReference })
 
   // TODO reproject default extents when non WGS84 CRS is found or passed
-
-  return _.defaults({
+  const overrides = {
     spatialReference,
     fullExtent,
     initialExtent: fullExtent,
```

Comment:
hello @brightgarden.  Thanks for this contribution. I think you can just add `copyrightText` here, and then you don't need to bother with the `if` block below.  `_.defaults` should handle it if undefined. I pulled down your branch, tested and confirmed.

Reply:
Done. I had handled it this way so I would not have to touch all those unit tests. It seemed like enabling a property override shouldn't require unit tests that are proxyquiring the defaults to list all the overridable properties.